src/client/components/login/login.component.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import './login.style.css';
+import PropTypes from 'prop-types';
+import {
+  faUser,
+  faKey,
+  faExclamationCircle,
+} from '@fortawesome/free-solid-svg-icons';
+
+import InputLogin from '../input-login/input-login.component';
+import Button from '../button/button.component';
+import Logo from '../logo/logo.component';
+import Link from '../link/link.component';
+import Notification from '../notification/notification.component';
+
+import imageFile from '../../assets/images/logo.png';
+
+export default function Login({
+  title,
+  errorNotifications,
+  handleChange,
+  loginFunc,
+}) {
+  const [errorMessage, setErrorMessage] = useState('');
+  useEffect(() => {
+    if (errorNotifications) {
+      if (errorNotifications === 'email')
+        setErrorMessage('This is error message for wrong email address');
+      else if (errorNotifications === 'password')
+        setErrorMessage('This is error message for wrong password.');
+      else setErrorMessage('Something went wrong. Try again later.');
+    }
+  }, [errorNotifications]);
```

Comment:
Looks like you're trying to make this a container component without it actually being the case.

You could make this a container, but if you do so you can't put it in storybook.

I think you're planning on calling the api in the useEffect hook, but I don't think it belongs here. I think there is yet another component above this one which will have the state. That component will be responsible for calling the API and passing data (or errors) from the API down to this component via props.

Remember the hierarchy I did:

    - LoginContainer (container)
      - Login (presentational, this component)
        - InputLogin
        - Button
        - etc

Now it looks like you are trying to do:

      - Login (container, this component)
        - InputLogin
        - Button
        - etc

But your container has props and `useEffect()` at the same time. If a container component calls an API it will usually get all of its state from there, not from props. There can still maybe be a few props related to theming or similar, but you would not have a component where you can give it data via props and it will also get data from an API call. If you do that the component will be functioning in very different ways when you use it in Storybook (with props) and in the real application (with an API call). The point of Storybook is to make as realistic an environmnet for the component as possible, but still isolated from the outside (isolated from APIs that may not be available and other side effects).